src/sidebar/jumpTo/flow/jumpToFlow.ts

```diff
@@ -64,7 +62,12 @@ async function startJumpToFlow() {
     // Show jumpable picker
     const jumpableQuickPickProvider = new QuickPickProvider(
         new JumpablesStore(barrelId, barrelType),
-        jumpable => getQuickPickItem(jumpable, barrel!),
+        jumpable => ({
+            label: jumpable.name,
+            detail: isComponent(jumpable)
+                ? getZeplinComponentDetailRepresentation(jumpable, true)
+                : getScreenDetailRepresentation(jumpable, barrel!)
```

Comment:
It would be better if we prefix `Zeplin` to `getScreenDetailRepresentation` or remove it from `getZeplinComponentDetailRepresentation` to be consistent. I'd prefer removing it.

Reply:
there 2 types of components in the application: codebase components and zeplin components. So we cannot remove Zeplin from this method and from many other places. For screens there is only one type of screen, so naming it zeplinScreen would be redundant. I would prefer all this stays the same for now. 

For the future, I plan to rename components so that "component" means zeplin component and "codebase component" means codebase component. This needs to thought to throughly but not right now.